### ISSUE TITLE
Training issues，labes.jpg Only the upper left corner has data, and the remaining three have no data

### ISSUE THREAD

**Issue by xiaolieat** [ultralytics/ultralytics#24324]
Hello, when I was training with YOLO, I used the following line of command for training, my dataset is a YOLO format dataset downloaded from kaggle, I don't know why after training, there is only the number of categories in the labels.jpg, and there is no data for the other three graphs

yolo train data=YOLO_format/data.yaml model=ultralytics/cfg/models/v8/yolov8n.yaml pretrained=yolov8n.pt epochs=100 batch=64 resume=True

Here's a picture from my labels.jpg
![Image](https://github.com/user-attachments/assets/95747f3b-b573-4d12-9f57-c18416ebdc59)

Here's my version of YOLO
![Image](https://github.com/user-attachments/assets/549be044-752c-445b-bb15-f0de9bcc3921)

Here's a configuration file for my dataset
![Image](https://github.com/user-attachments/assets/10f8168f-e818-4acc-9e4a-fa695b6e12a0)

**Comment by UltralyticsAssistant** [comment]
👋 Hello @xiaolieat, thank you for reporting this and for sharing screenshots 🚀 This is an automated response to help gather the right debugging details, and an Ultralytics engineer will also assist you soon.

We recommend a quick visit to the [Docs](https://docs.ultralytics.com/) where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples, and where many common training and dataset questions may already be answered.

Since this appears to be a 🐛 training/data visualization issue, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it. For this type of issue, helpful details include:
- The exact training command you ran
- The full console log from training start through dataset scanning
- A few sample images and their corresponding YOLO label `.txt` files
- The contents of your `data.yaml` as text
- Confirmation that the issue still occurs after upgrading to the latest package

If this is a custom training ❓ question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.
